src/Kralizek.Extensions.Configuration.AWSSecretsManager/Internal/SecretsManagerConfigurationProvider.cs

```diff
@@ -258,6 +267,133 @@ private async Task<IReadOnlyList<SecretListEntry>> FetchAllSecretsAsync(Cancella
             }
             return configuration;
         }
+        
+        private static List<List<SecretListEntry>> ChunkList(IReadOnlyList<SecretListEntry> source,
+            Func<SecretListEntry, bool> optionsSecretFilter, int chunkSize)
+        {
+            // This is for sake of cleanliness vs getting 'fancy' with things.
+            // We can always optimize later.
+            return source
+                .Where(sle=> !optionsSecretFilter(sle))
```

Comment:
By default, SecretFilter is set to always return `true`, So assuming it is meant to be a positive filter, The `!` operator here prevents those secrets from passing. I.e. on a default setup, this drops all secrets and return an empty list.

Reply:
I derped hard on this one, my bad. Fixing.

Reply:
Don't we all. No worries :]